assets/reset.scss

```diff
@@ -34,7 +34,48 @@
 .narrow-scrollbar .simplebar-track {
   width: 8px !important;
 }
+.wide-scrollbar .simplebar-track {
+  width: .7em !important;
+  & .simplebar-scrollbar:before {
+    left: 0 !important;
+    right: 0 !important;
+    top: 0 !important;
+    bottom: 0 !important;
+  }
+  & .simplebar-track {
+    background: transparentize(map-get($greys, "100"), 0.5) !important;
+    border-radius: .7em;
+  }
+}
 
 b {
   font-weight: 600 !important;
 }
+
+.simplebar {
+  min-height: 0;
```

Comment:
If this should be the default (it's sprinkled on every `<simplebar/>`), why does it have to be explicitly added?